.github/main.workflow

```diff
@@ -7,3 +7,14 @@ action "verify-emoji" {
     uses = "./hack/release"
     secrets = ["GITHUB_TOKEN"]
 }
+
+workflow "Linters and Test" {
+    on = "push"
+    resolves = ["lint"]
+}
+
+action "lint" {
```

Comment:
Why is this done via github actions and not via  a prow presubmit like the other tests?

Reply:
It is called by the prow already. See that if it is missing some comment the test will fail. 

Example: 

```
level=info msg="[runner] linters took 9.994443697s with stages: goanalysis_metalinter: 9.9921147s"
pkg/internal/testing/integration/internal/etcd.go:7:1: comment on exported var `EtcdDefaultArgs` should be of the form `EtcdDefaultArgs ...` (golint)
//todo(just to ensure that the pr will fails and the test is called in the prow): EtcdDefaultArgs allow tests to run offline, by preventing API server from attempting to
```

ref: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_controller-runtime/885/pull-controller-runtime-test-master/1247116818334617600/build-log.txt

Test executed just to ensure that it will fail.